app/views/layouts/kadmin/application.html.erb

```diff
@@ -97,7 +97,12 @@
     <%= javascript_include_tag "kadmin/application" %>
     <script type='text/javascript'>
       jQuery(document).ready(function () {
-        jQuery('#sidebar-menu').metisMenu({ activeClass: 'open' });
+          jQuery('#sidebar-menu').metisMenu({ activeClass: 'open' });
```

Comment:
I know it's not ruby but js, but let's keep the same identation (i.e. 2 spaces) rule for all the code.